MIGRATION.md

```diff
@@ -2,6 +2,8 @@
 
 ### Migrating from v1 to v2
 
+`o-normalise` now depends on `o-colors`. Ensure your project builds successfully and upgrade `o-colors` if there is a conflict.
```

Comment:
I think we should state it is specifically o-colors version ^4.10.1 that it should use.

Reply:
This will be released as part of the major breaking cascade, so we'll have another pass at this when we bump to `o-colors@5.0.0` :)